pkgs/by-name/ni/nixfmt-tree/package.nix

```diff
@@ -62,7 +80,7 @@ buildEnv {
         ```
 
         You can then also use `treefmt` in a pre-commit/pre-push [Git hook](https://git-scm.com/docs/githooks)
-        and `nixfmt` with your editors format-on-save feature.
+        and with your editor's format-on-save feature.
```

Comment:
Yeah so this was the original motivation for including the formatter binaries as well, but as explained in the last formatting team meeting, people should be using treefmt even for format-on-save, because treefmt knows about the options you format the codebase with :)